floto/api/tasks.py

```diff
@@ -95,3 +99,25 @@ def rename_devices():
             if labelname and labelname != device_name:
                 LOG.info(f"setting device name from {device_name} to {labelname}")
                 balena.models.device.rename(device_uuid,labelname)
+
+
+@shared_task(name="bulk_device_update_CSV")
+def bulk_device_update_CSV(file):
+    reader = csv.DictReader(file)
+    for row in reader:
+        device = DeviceData.objects.get(device_uuid=row["device_uuid"])
```

Comment:
I think this should have error handling, so that it'll import as many as possible. Right now if an exception happens, the file will only be partially imported. Can it print an error message on exception as well?